vck/src/commonMain/kotlin/at/asitplus/wallet/lib/agent/CryptoService.kt

```diff
@@ -60,13 +60,29 @@ interface CryptoService {
     fun hmac(
         key: ByteArray,
         algorithm: JweEncryption,
-        input: ByteArray
+        input: ByteArray,
     ): KmmResult<ByteArray>
 
     val keyMaterial: KeyMaterial
 
 }
 
+typealias VerifySignatureFun = (
```

Comment:
If this is a function now that just delegates to another function (and the `SignatureInput` not being needed any more with the properly published Android artefact of supreme), why the drastically different function signature compared to Supreme?. I get the quality-of-life wrapper, but why not an extension like
`X509SignatureAlgorithm.verify(input: ByteArray, signature: X509Signature, publicKey: CryptoPublicKey)`

on a related note: do we really want X509SignatureAlgorithms here, or the more geneic `SignatureAlgorithm`. Since we're already refactoring…

Reply:
Because I'd like to keep the changes to calling code minimal. But the suggestion with using `SignatureAlgorithm` is good, I'll change that!